stream.go

```diff
@@ -96,14 +96,33 @@ type ClientStream interface {
 // NewClientStream creates a new Stream for the client side. This is called
 // by generated code.
 func NewClientStream(ctx context.Context, desc *StreamDesc, cc *ClientConn, method string, opts ...CallOption) (ClientStream, error) {
+	var (
+		conn *Conn
+		t    transport.ClientTransport
+		err  error
+	)
+	for {
+		conn, err = cc.picker.Pick()
+		if err != nil {
```

Comment:
Shouldn't this backoff and retry if the picker error is transient?  (Or is Pick supposed to block - in which case why doesn't Pick take a Context?)